### PR TITLE
Fix #143: prevent searching empty field

### LINKED ISSUE #143: Search  ( SearchInput.js )
I don't have skills in react so I can't really contribute to the fix,  but when clicking search and the input box is empty it will say: "-0 OF 0 SEARCH RESULTS FOR:"  and say "loading"  forever...
Maybe make the input "glow"/shake/focus  if it is empty? :D

### PR DESCRIPTION
**Pull request by reshadn**
Also formatting cleanup.

### CODE REVIEW

File: src/scripts/components/header/SearchInput.js [hunk 1]
```diff
@@ -1,69 +1,85 @@
...
+    search(e) {
+        if (e.key === 'Enter' || e.keyCode === 13 || e.type === 'click') {
+            let inputEl = React.findDOMNode(this).querySelector('input'),
+                searching = inputEl.value.trim(),
+                ayah, pattern, surah;
 
-      const shortcutSearch = /\d[\.,\:,\,,\\,//]/g;
-      const splitSearch = /[\.,\:,\,,\\,//]/g;
+            // prevent search function while search input field is empty
+            if (searching === '') {
```
Comment (reshadn) [inline src/scripts/components/header/SearchInput.js]: @mmahalwy This is the #143 fix. I hope you don't mind formating cleanup. Do we have a standard for formatting files? Suggetions?

Comment (mmahalwy) [inline src/scripts/components/header/SearchInput.js]: We use eslint for linting our JS which should have thrown an error or tests failing for this indentation (I will change it) but I am implementing 2 spaces indentation through-out. Following Airbnb's js: https://github.com/airbnb/javascript

I like 2 spaces more than 4 cause reactjs being its a blend of html & js could get really really messy really fast with 4 spaces and a pain to code review, hence 2 spaces :)

**Comment by reshadn** [comment]
Sorry for all the formatting issues. Should be back to normal now.

**Comment by mmahalwy** [comment]
No worries! Looking good!

**Comment by mmahalwy** [comment]
@reshadn merge when ready

**Pull request merged**